description: adjusting order of dependent libraries in setup.py
issue: https://github.com/easybuilders/easybuild-easyconfigs/issues/14219
author: Zdenek Matej, MAX IV Laboratory, Lund University
--- setup.py
+++ setup.py
@@ -77,7 +77,7 @@
 }
 
 
-libraries = ['png', 'jpeg', 'tiff']
+libraries = ['png', 'tiff', 'jpeg']
 if sys.platform.startswith('win'):
     libraries.append('zlib')
 
